client/components/NavBar.jsx

```diff
@@ -77,7 +78,16 @@ export default function NavBar() {
             </NavItem>
             <NavItem>
               <Col className="button-col">
-                <Button className="button-create" color="#ffcc66">
+                <Button
+                  className="button-create"
+                  onClick={function handleClick() {
```

Comment:
Use an arrow function here to maintain some consistency. Also, it might be better to move this function to the js section above the return and then call it here from a stylistic standpoint.